### PR TITLE
SM-7596 Docs update for SM-7596

### PR DESCRIPTION
**Pull request by MatthewDoherty994**

### CODE REVIEW

File: docs/api-documentation/versions-and-changes/v3/changelog.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ Updated Work API with the following changes:
   <li>
     SM-6511: Added <code>DELETE /works/{workReferenceNumber}/sites/{siteReferenceNumber}/reinstatements/{reinstatementReferenceNumber}</code> to allow the deletion of reinstatements created in error. Two new audit events will be returned in the work history <code>reinstatement_deleted</code> and <code>site_deleted</code> to record the deletions. Reference numbers of deleted sites/reinstatements will never be re-used, any subsequently created site/reinstatement will not take the reference number of the previously deleted site/reinstatement. Any inspections with non compliant sites will still show that the deleted site was non-compliant.
   </li>
+  <li>
+    SM-7596: Updated the <code>GET /works/{workReferenceNumber}/section-74s/{section74ReferenceNumber}</code> endpoint to only return fields relavant to the
+    HA and Promoter stauses.
```
Comment (lukeca362) [inline docs/api-documentation/versions-and-changes/v3/changelog.md:26]:
```suggestion
    HA and Promoter statuses.
```

File: docs/api-documentation/versions-and-changes/v3/changelog.md [hunk 2]
```diff
@@ -21,6 +21,10 @@ Updated Work API with the following changes:
   <li>
     SM-6511: Added <code>DELETE /works/{workReferenceNumber}/sites/{siteReferenceNumber}/reinstatements/{reinstatementReferenceNumber}</code> to allow the deletion of reinstatements created in error. Two new audit events will be returned in the work history <code>reinstatement_deleted</code> and <code>site_deleted</code> to record the deletions. Reference numbers of deleted sites/reinstatements will never be re-used, any subsequently created site/reinstatement will not take the reference number of the previously deleted site/reinstatement. Any inspections with non compliant sites will still show that the deleted site was non-compliant.
   </li>
+  <li>
+    SM-7596: Updated the <code>GET /works/{workReferenceNumber}/section-74s/{section74ReferenceNumber}</code> endpoint to only return fields relavant to the
```
Comment (lukeca362) [inline docs/api-documentation/versions-and-changes/v3/changelog.md:25]:
```suggestion
    SM-7596: Updated the <code>GET /works/{workReferenceNumber}/section-74s/{section74ReferenceNumber}</code> endpoint to only return fields relevant to the
```

File: docs/api-documentation/versions-and-changes/v3/changelog.md [hunk 3]
```diff
@@ -21,6 +21,10 @@ Updated Work API with the following changes:
...
+    HA and Promoter stauses.
+  </li>
```
Comment (ccurran01) [inline docs/api-documentation/versions-and-changes/v3/changelog.md:27]:
```suggestion
  <li>
    SM-7596: Updated the <code>GET /works/{workReferenceNumber}/section-74s/{section74ReferenceNumber}</code> endpoint to only return fields relavant to the current
    HA and Promoter stauses.
  </li>
```

**Review by lukeca362 — CHANGES_REQUESTED** [review]
Just a few comments. And maybe move to it all on to one line?

**Review by ccurran01 — COMMENTED** [review]
There is a section in the V3 resource guide which will be out of date after we merge this story. That will also need an update to better reflect what is returned and when

**Review by lukeca362 — APPROVED** [review]

**Review by ccurran01 — APPROVED** [review]

**Pull request merged**